src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useEffect, useState } from "react";
+import { Selector } from "./components/Selector";
+import { CalculatorWindow } from "./components/CalculatorWindow";
+import { stateTable } from "./stateData";
+import { ComparisonOptions } from "./apps/RatioComparer/ComparisonOptions";
+import Player from "./apps/Player/Index";
+import TopBar from "./components/TopBar";
+import About from "./components/About";
+import { playerStateData } from "./apps/Player/playerStateData";
+import BottomBar from "./components/BottomBar";
 
 function App() {
+  const [stateData, setStateData] = useState(stateTable);
+  const [playerState, setPlayerState] = useState(playerStateData);
+
+  // useEffect(() => {
+  //   console.log({ stateData });
+  // }, [stateData]);
+
+  const [width, setWidth] = useState();
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth - 1);
+      setWidth(window.innerWidth);
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    return (_) => {
+      window.removeEventListener("resize", handleResize);
+    };
```

Comment:
Unless I'm missing something (always a possibility) it looks like this does nothing? The width is set on resize but then the app doesn't do anything with it.